### ISSUE TITLE
php: command not found

### ISSUE THREAD

**Issue by fabston** [HilbertGilbertson/alfred-snowcord#1]
Hi there,

I installed snowcord but unfortunately I am getting the following error when using it:

```
Code 127: /Users/fs/Library/Caches/com.runningwithcrayons.Alfred/Workflow Scripts/2BC30EB7-4AA4-446F-B50E-858C993FDD68: line 2: php: command not found
```

Do you have an idea why that is happening?

Appreciate your help!

**Comment by HilbertGilbertson** [comment]
I'm assuming this is with the new v1.0.1 released last night (and not 1.0.0)?

When Snowcord runs it uses bash to execute the core php file (Alfred _can_ store scripts directly in the workflow, but this means bundling them into Info.plist rather than keeping them file based, which is simpler to maintain), but in your case bash can't find the php binary. Previously (in 1.0.0) I'd sourced ~/.bashrc to get the Homebrew environment path, which is quite a bit of a legacy (it used to be created by Homebrew but no longer as the default shell changed from bash to zsh back in MacOS Catalina), but in any case it didn't matter if it failed because stock php was still in MacOS ... unless you were on Monterey.

Since last night's 1.0.1 update, Snowcord will check whether `/opt/homebrew/bin/brew` exists, and if so source the environment from `/opt/homebrew/bin/brew shellenv` (via same method in Whencord 1.0.1, which I'm glad to hear is working for you). This is so that for pre-Monterey users, stock php will be used as a fallback, while any version installed with Homebrew can take precedence. Monterey should also now run Snowcord without issue, providing PHP is installed via Homebrew.

I was unable to replicate this on my fresh MacOS Monterey demo install. Do let me know if you're running 1.0.1

**Comment by HilbertGilbertson** [comment]
@fabston Do let me know if you're still experiencing this issue, or whether it was resolved by 1.0.1.

I'll leave this open a couple more days either way.

**Comment by HilbertGilbertson** [comment]
I presume this was resolved so I'm closing this issue. Do feel free to reply if you're still experiencing the problem.
